src/pokemon.ts

```diff
@@ -0,0 +1,13 @@
+export class Pokemon {
+    constructor(name: string, hitpoints: 30) {
```

Comment:
o tipo do hitpoints poderia ser number
```suggestion
    constructor(name: string, hitpoints: number) {
```